extension/src/background.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * GPII Chrome Extension for Google Chrome
+ *
+ * Copyright 2016 RtF-US
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this license.
+ *
+ * You may obtain a copy of the license at
+ * https://github.com/GPII/gpii-chrome-extension/blob/master/LICENSE.txt
+ */
+
+/* global fluid */
+
+"use strict";
+
+fluid.setLogging(true);
+var gpii = fluid.registerNamespace("gpii");
+var chromeSettings = gpii.chrome.settings();
+
+fluid.log("Instantiated chromeSettings:", chromeSettings);
```

Comment:
I imagine this doesn't log anything very intelligible as the value of the component